modules/services/blueman-applet.nix

```diff
@@ -13,7 +13,7 @@ with lib;
           be enabled system-wide. You can enable it in the system
           configuration using
           <programlisting language="nix">
-            services.blueman.enable = true;
+            services.blueman-applet.enable = true;
```

Comment:
Are you sure this is correct? My understanding of the description is that for

    services.blueman-applet.enable = true;

to actually work for your HM configuration you need to have

    services.blueman.enable = true;

in your NixOS configuration. Searching for "blueman" among the NixOS configuration options shows no blueman-applet module: https://search.nixos.org/options?query=blueman

Reply:
No, I'm wrong. Sorry for the spam.

Reply:
No worries.